src/utils.rs

```diff
@@ -36,8 +36,13 @@ pub fn hexstring_to_vec(hexstring: &str) -> Result<Vec<u8>, SimulationError> {
 }
 
 /// Loads all tokens from Tycho and returns them as a Hashmap of address->Token.
-pub async fn load_all_tokens(tycho_url: &str, auth_key: Option<&str>) -> HashMap<Bytes, Token> {
-    let rpc_url = format!("https://{tycho_url}");
+pub async fn load_all_tokens(
```

Comment:
I wonder if it makes sense or if we shouldn't just give `tycho_url` with the prefix already there?

Reply:
I think no, because the tycho url is also used in more places and the prefixes are added there as well :/ or do you mean to add the prefix only once in a centralized place?